### ISSUE TITLE
order by year not working in 'my subscriptions' tab

### ISSUE THREAD

**Issue by voidplayer** [bromix/plugin.video.youtube#327]
kodi 15

there is no way to order by more recent because dates on the videos are all 1980

not sure you are interested in this bug reading #313 but maybe is easy to fix in the meantime ;)

**Comment by bromix** [comment]
It will cripple the pagination. That's why no sorting is provided.

**Comment by voidplayer** [comment]
well, it worked in kodi 14

When I go to 'my subscriptions' they were ordered by 'last uploaded'... now my subscription shows everything ordered by name, since the date in all the videos is the same

The year shown for all videos is 1980. I cant remember the exact day but i can look it up if is important :)

the fact that the date of publishing is wrong makes me think is a bug... (also the fact that it worked in kodi 14)

**Comment by voidplayer** [comment]
![ssvnc wonderland 0 - press f8 for menu_166](https://cloud.githubusercontent.com/assets/4019910/9909419/6dff8f68-5c98-11e5-8711-2be880dcc524.png)

they are ordered by year in this pic. Actually the date of all videos changed from 1980 today for some reason

what method should I use if I want to see the latest videos first?

**Comment by bromix** [comment]
You've your sorting is...change it to descending.
Navigate to left and change it in the sidebar

**Comment by voidplayer** [comment]
ascending or descending is the same... since the date is wrong. I tried both :)

The first page of the pagination is the only thing that gets ordered (meaning, i dont get older videos by ordering descending)

I have a favorite to my subscription from the main menu and thats how i access 'my subscritions', but I tried accessing it from the addon menu and it magically started worked again...

Maybe the dates dont get updated when accessing via favorite or something. I dont know... new videos did showed up... just unordered. It happened since I upgraded from kodi 14 :)

Anyway, its working now!
thanks!
